icubam/messaging/handlers/onoff_test.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+from icubam.messaging.handlers import onoff
+
+class OnOffRequestTestCase(unittest.TestCase):
```

Comment:
Since tests are run with pytest, you don't need importing `unittest`:

```suggestion
class OnOffRequestTestCase:
```